onboard/import.go

```diff
@@ -62,7 +67,12 @@ func CreateImporter(ctx context.Context, logger logging.Logger, config *Config)
 		return nil, fmt.Errorf("failed to get previous commit: %w", err)
 	}
 	res.previousCommit = previousCommit
-	res.inventory, err = config.InventoryGenerator.GenerateInventory(ctx, logger, config.InventoryURL, res.previousCommit != nil)
+	shouldSort := res.previousCommit != nil
+	res.inventory, err = config.InventoryGenerator.GenerateInventory(ctx, logger, config.InventoryURL, shouldSort, config.KeyPrefixes)
+	res.prefixes = config.KeyPrefixes
+	if !res.validateSamePrefixes() {
```

Comment:
why do we care?

Reply:
synced f2f: will validate that the new set of prefixes contains the previous one